### DISCUSSION TITLE
Getting Map Information

### DISCUSSION THREAD

**Discussion by Kai-X-Org** [HumanSupportRobot/QandA#82] [Q&A]
Hi,

Is there a way to obtain information on the map that the simulation is run in (what kind of objects included, positions of objects dimensions, etc.)? Is there a file that documents all this? Thank you so much!

**Comment by yuka-hashiguchi** [comment]
Would it be possible for you to obtain the information you are seeking from the following?

Firstly, let's verify the world file being loaded in the simulator's startup script. For instance, if you are launching the simulator using Command `roslaunch hsrb_gazebo_launch hsrb_megaweb2015_world.launch`, you should examine File `/opt/ros/noetic/share/hsrb_gazebo_launch/launch/hsrb_megaweb2015_world.launch`. This will provide you with insights into the map and gazebo world model information being loaded from the paths configured in variables like "map" and "world_name." The world file is located under path `/opt/ros/noetic/share/tmc_gazebo_worlds/worlds`, so please take a look at its contents.

Please be aware that the copyright for these codes is owned by TMC, so there are limitations on their public dissemination.
